controllers/syncer/csi_node.go

```diff
@@ -83,10 +83,10 @@ func (s *csiNodeSyncer) SyncFn(daemonSetRestartedKey string, daemonSetRestartedV
 
 	// ensure template
 	out.Spec.Template.ObjectMeta.Labels = nodeLabels
-	out.Spec.Template.ObjectMeta.Annotations = s.driver.GetAnnotations(daemonSetRestartedKey, daemonSetRestartedValue)
+	controllerAnnotations := s.driver.GetAnnotations(daemonSetRestartedKey, daemonSetRestartedValue)
```

Comment:
can you rename `controllerAnnotations` to `nodeAnnotations` in this file?

Reply:
Thanks, renaming